src/NuGet.Core/NuGet.PackageManagement/Utility/PackagesConfigLockFileUtility.cs

```diff
@@ -55,10 +55,6 @@ public static class PackagesConfigLockFileUtility
                     var projectUri = new Uri(msbuildProject.MSBuildProjectPath);
                     var lockFileUri = new Uri(lockFileName);
                     var lockFileRelativePath = projectUri.MakeRelativeUri(lockFileUri).OriginalString;
-                    if (Path.DirectorySeparatorChar != '/')
```

Comment:
This code is only exercised for PC projects with lockfile enabled( or lock file already exist) and new package is installed or existing one is upgraded.
According to this [comment](https://github.com/NuGet/NuGet.Client/pull/2492#discussion_r249950162) there was supposed to be  `LockFileNamePath` property to put the lock file in a directory other than where the csproj is (either a sub-directory or a parent directory). But it looks `LockFileNamePath` not implemented as of today, therefore `lockFileRelativePath` would be just filename(`packages.lock.json`), so removed code is technically not relevant.

Reply:
`LockFileNamePath` may not be implemented but [`NuGetLockFilePath`](https://github.com/NuGet/NuGet.Client/search?q=NuGetLockFilePath) is implemented. As per https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets doc, `NuGetLockFilePath` points to `a custom location for the lock file. The default location is next to the project and is named packages.lock.json.` 

I found a bug when trying to exercise a customer scenario that executes this code. https://github.com/NuGet/Home/issues/11149

`lockFileRelativePath` is passed as a parameter to `AddExistingFile` method which is relevant in only VS scenarios https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectSystems/VsMSBuildProjectSystem.cs#L233. In `nuget.exe` the said method has no implementation. https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.CommandLine/Common/MSBuildProjectSystem.cs#L101

Given that this scenario is currently broken in VS, I am guessing it is fine to remove this code.